.github/CODEOWNERS

```diff
@@ -0,0 +1,22 @@
+# The purpose of this file is to trigger review requests when PRs touch
+# particular files. Those reviews are not mandatory, however it's often useful
+# to have an expert pinged who is interested in only one part of MNE and
+# doesn't follow general development.
```

Comment:
perhaps it'd be worth to mention that - despite the name of this file - having **your** name in this file does not really imply "ownership", but only that you want to be pinged?